### PR TITLE
Create jian-css.json

### PR DESCRIPTION
**Pull request by XYZscratcher**
### Requirements
- [x] You're not using Vercel or Netlify.
- [x] You have completed your website, there's no type of placeholder at the website. *This requirement is optional if you are registering a domain for emails.*
- [x] The website is reachable.
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.
   - *You should have your email presented. If you don't want to share email, you can leave email an empty string (`""`) and add any other social such as Discord/Twitter/etc.*

### CODE REVIEW

File: domains/jian-css.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "一个超级轻的 CSS 框架。",
+    "repo": "https://github.com/jian-css/jian-css.github.io",
+    "owner": {
+        "username": "jian-css",
+        "email": "email@address",
+        "twitter": null
```
Comment (WilliamDavidHarrison) [inline domains/jian-css.json:7]:
```suggestion
```

File: domains/jian-css.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "一个超级轻的 CSS 框架。",
+    "repo": "https://github.com/jian-css/jian-css.github.io",
+    "owner": {
+        "username": "jian-css",
+        "email": "email@address",
```
Comment (WilliamDavidHarrison) [inline domains/jian-css.json:6]:
```suggestion
        "email": ""
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/jian-css.json (hunk 2 above)
Comment (XYZscratcher) [reply domains/jian-css.json]: Thanks for the reminder!

**Review by wdhdev — CHANGES_REQUESTED** [review]
The website needs to be mainly english, sorry.

**Pull request closed without merging**